packages/atlas/src/views/global/YppLandingView/YppCardsSection.styles.ts

```diff
@@ -40,6 +40,7 @@ export const CardImage = styled.img<{ absolute?: boolean; dropShadow?: boolean }
   width: 100%;
   min-width: 480px;
   max-width: 640px;
+  height: auto;
```

Comment:
Why do we need to set `height: auto`. This is the default value.

Reply:
It's not when we set it as HTML attribute https://github.com/Joystream/atlas/pull/3335/files#diff-fb3a251bc61562a12fc7ffb4834ee80ed8ce21f63ef798ec792da761abfb3dd3R86